overlord/ifacestate/ifacemgr.go

```diff
@@ -156,6 +159,38 @@ func (m *InterfaceManager) doSetupSnapSecurity(task *state.Task, _ *tomb.Tomb) e
 	return nil
 }
 
+type connState struct {
+	Auto      bool   `json:"auto,omitempty"`
+	Interface string `json:"interface,omitempty"`
```

Comment:
Can we please name this one as "iface" on the JSON side?  We'll probably have a large number of these, so it won't hurt to use the well-accepted short name.